pandas/tests/io/test_stata.py

```diff
@@ -1848,6 +1847,29 @@ def test_backward_compat(version, datapath):
     tm.assert_frame_equal(old_dta, expected, check_dtype=False)
 
 
+def test_direct_read(datapath, monkeypatch):
+    file_path = os.path.join(datapath("io", "data", "stata"), f"stata-compat-118.dta")
```

Comment:
I think (can't test it right now), that you can shorten this to
`file_path = datapath("io", "data", "stata", "stata-compat-118.dta")`

Reply:
Yep, my bad. I had copy-pasted and adapted that from the test above.